hera_mc/mc_session.py

```diff
@@ -2377,6 +2378,11 @@ def correlator_control_command(self, command, starttime=None, duration=None,
             else:
                 if acclen_spectra != corr.DEFAULT_ACCLEN_SPECTRA:
                     warnings.warn('Using a non-standard acclen_spectra!')
+
+            if (not isinstance(acclen_spectra, (int, np.integer))
+                    or np.round(acclen_spectra / 2048.) * 2048 - acclen_spectra != 0):
```

Comment:
Is there a reason this can't just be (acclen_spectra % 2048 != 0)?

Reply:
No, I'll change it.